Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/DXResourceBindingORT/DXResourceBindingORT.xaml.cs

```diff
@@ -0,0 +1,124 @@
+using Microsoft.AI.MachineLearning;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Windows.Graphics.Imaging;
+using Windows.Media;
+using Windows.Storage;
+using WinMLSamplesGalleryNative;
+using System.Linq;
+
+namespace WinMLSamplesGallery.Samples
+{
+    public class DXClassifierResult
+    {
+        public int index { get; private set; }
+        public string label { get; private set; }
+        public DXClassifierResult(int i, string lab)
+        {
+            index = i;
+            label = lab;
+        }
+    }
+
+    public sealed partial class DXResourceBindingORT : Page
+    {
+        private bool pageExited = false;
+
+        public DXResourceBindingORT()
+        {
+            this.InitializeComponent();
+
+            // Run in background thread to avoid blocking UI on sample launch
+            Task.Run(() => SetModelNameForTelemetry());
+        }
+
+        private async void LaunchWindow(object sender, RoutedEventArgs e)
+        {
+            ToggleLaunchWindowButton();
+            ToggleLoadingTxtVisibility();
+
+            await Task.Run(() => WinMLSamplesGalleryNative.DXResourceBinding.LaunchWindow());
+
+            ToggleLoadingTxtVisibility();
+            ClassifyCube();
+        }
+
+        private async void ClassifyCube()
+        {
+            int i = 0;
+            float[] prev_results = { };
```

Comment:
```suggestion
previousResults
```

*Fully spelling out words, plus camelCase consistent with standard C# and .NET conventions (and the existing `pageExited`).

Reply:
Done.